components/normalizer/src/lib.rs

```diff
@@ -67,6 +67,18 @@
 
 extern crate alloc;
 
+// We don't depend on icu_properties to minimize deps, but we want to be able
+// to ensure we're using the right CCC values
+macro_rules! ccc {
+    ($name:ident, $num:expr) => {{
+        #[cfg(all(debug_assertions, feature = "icu_properties"))]
```

Comment:
wouldn't even gate this on debug_assertions

Reply:
I'm using it in non const contexts too. I could make consts for everything

Reply:
You don't need to define consts, the macro can produce a const expression:

```rust
{ 
  const X: u8 = {
    #[cfg(all(debug_assertions, feature = "icu_properties"))]
    if icu_properties::CanonicalCombiningClass::$name.0 != $num {
      panic!("icu_normalizer has incorrect ccc values")
    }
    $num
  };
  X
}
```